models/Translation/Dao.php

```diff
@@ -105,7 +105,7 @@ public function save(): void
                         'key' => $this->model->getKey(),
                         'type' => $this->model->getType(),
                         'language' => $language,
-                        'text' => $sanitizer->sanitize($text),
+                        'text' => html_entity_decode($sanitizer->sanitize($text)),
```

Comment:
this can be removed https://github.com/pimcore/admin-ui-classic-bundle/blob/9e9ce75c0f4d8ff6db591b1bf6539dac930f6d87/src/Controller/Admin/TranslationController.php#L377 right?

Reply:
yes, https://github.com/pimcore/admin-ui-classic-bundle/issues/275 but not sure because the exact line matches with a security fix PR